package.json

```diff
@@ -67,12 +69,13 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@centreon/ui": "centreon/centreon-ui",
+    "@centreon/ui": "github:centreon/centreon-ui",
```

Comment:
why this change?

Reply:
it's automatically added since I've upgraded npm